scripts/test-visual.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Exit the script on any command with non 0 return code
+set -e
+
+# Go to project root
+cd "$(dirname "$0")"
+cd ..
+
+export ECL_SYSTEM=ec
+
+TEST_BROWSER=chrome npx jest test/visual.test.js
```

Comment:
when running `yarn test:visual`, test are run only on chrome, not sure why

Reply:
Indeed, I have to think of parallel-izing this, as a failure stops the pipeline.

Reply:
please use `yarn` instead of `npx` :)

Reply:
we might not need a separated script anyway, those are only npm commands in the end, we could write them in the `package.json` file instead, with https://www.npmjs.com/package/cross-env and https://www.npmjs.com/package/npm-run-all it's quite easy to run the tests in parallel or sequentially ;)

```json
"test:visual": "npm-run-all --parallel test:visual:*",
"test:visual:ec": "cross-env ECL_SYSTEM=ec npm-run-all --parallel watch:*",
"test:visual:chrome": "cross-env TEST_BROWSER=chrome jest test/visual.test.js",
...
"test:visual:eu": "cross-env ECL_SYSTEM=eu npm-run-all --parallel watch:*",
...
```